### PR TITLE
Fix the program crashing when confirming an invalid setting value

### LINKED ISSUE #669: Crash when confirming empty setting
GodSVG will crash because of NAN value when having empty settings for ui scale or handle size.

### PR DESCRIPTION
**Pull request by ilikefrogs101**
Fixes #669

### CODE REVIEW

File: src/parsers/NumberParser.gd [hunk 1]
```diff
@@ -50,17 +50,17 @@ static func evaluate(text: String) -> float:
 	var expr := Expression.new()
 	var err := expr.parse(text.replace(",", "."))
 	if err == OK:
-		var result: float = expr.execute()
-		if not expr.has_execute_failed():
-			return result
+		var result: String = var_to_str(expr.execute())
+		if not expr.has_execute_failed() and result.is_valid_float():
+			return str_to_var(result);
```
Comment (MewPurPur) [inline src/parsers/NumberParser.gd:55]: Let's not have semicolons.

Comment (ilikefrogs101) [reply src/parsers/NumberParser.gd:55]: Oops, lmao. every single time. I cant avoid it apparently

Comment (ilikefrogs101) [reply src/parsers/NumberParser.gd]: Fixed

**Comment by MewPurPur** [comment]
Thanks!

**Pull request merged**